docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -62,30 +57,54 @@ <h5>
     <p>Google Earth Enterprise 5.3.1 is compatible with Google Earth Enterprise Client (EC) version 7.1.5 and above.</p>
     <p><p>To upgrade from Open GEE 5.2.0, do NOT uninstall it. We recommend that you upgrade Open GEE 5.2.0 by simply installing Open GEE 5.3.1. Installing Open GEE 5.3.1 on top of Open GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by Open GEE 5.3.1.</p></p>
   <h5>
-    <h5>
-      <p id="resolved_issues_5.3.1">Resolved Issues</p>
-    </h5>
+    <h3 id="resolved_issues_5.3.1">Resolved Issues</h3>
```

Comment:
There is still an h3 tag inside an h5 tag. Please fix as above

Reply:
Removed